Makefile error: Directory examples/sw/led no longer present

## Observed Behavior
I am not sure the provenance of the Makefile in this project, maybe it is deprecated.  However I am trying to build a design arty the arty100 and it bombs out when that target calls the other target sw-led.

```
cd examples/sw/led && make
/bin/sh: 1: cd: can't cd to examples/sw/led
make: *** [Makefile:65: examples/sw/led/led.vmem] Error 2
```
Checking the contents of the base directory there is no `examples/sw/led` present, i checked back in time and it was in the past.

## My Environment

Ubuntu and python 3.12 under venv, but this is a basic Makefile error

For clarity the Makefile should be fixed or removed.  Is it supposed to be possible to build a working arty7 system from this repo any more or is this only maintained in [ibex-demo-system](https://github.com/lowRISC/ibex-demo-system) now?

--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ help:
 	@echo "or how to set-up the different environments."
 
 # Use a parallel run (make -j N) for a faster build
-build-all: build-riscv-compliance build-simple-system build-arty-100 \
+build-all: build-riscv-compliance build-simple-system \
       build-csr-test
 
 
@@ -50,34 +50,6 @@ $(Vibex_simple_system):
 run-simple-system: sw-simple-hello | $(Vibex_simple_system)
 	build/lowrisc_ibex_ibex_simple_system_0/sim-verilator/Vibex_simple_system \
 		--raminit=$(simple-system-program)
-
-
-# Arty A7 FPGA example
-# Use the following targets (depending on your hardware):
-# - "build-arty-35"
-# - "build-arty-100"
-# - "program-arty"
-arty-sw-program = examples/sw/led/led.vmem
-sw-led: $(arty-sw-program)
-
-.PHONY: $(arty-sw-program)
-$(arty-sw-program):
-	cd examples/sw/led && $(MAKE)
-
-.PHONY: build-arty-35
-build-arty-35: sw-led
-	fusesoc --cores-root=. run --target=synth --setup --build \
-		lowrisc:ibex:top_artya7 --part xc7a35ticsg324-1L
-
-.PHONY: build-arty-100
-build-arty-100: sw-led
-	fusesoc --cores-root=. run --target=synth --setup --build \
-		lowrisc:ibex:top_artya7 --part xc7a100tcsg324-1
-
-.PHONY: program-arty
-program-arty:
-	fusesoc --cores-root=. run --target=synth --run \
-		lowrisc:ibex:top_artya7
 
 
 # Lint check

